src/Libraries/Microsoft.Extensions.TimeProvider.Testing/FakeTimeProvider.cs

```diff
@@ -17,20 +17,9 @@ namespace Microsoft.Extensions.Time.Testing;
 public class FakeTimeProvider : TimeProvider
 {
     internal readonly List<FakeTimeProviderTimer.Waiter> Waiters = new();
-
-    private DateTimeOffset _now;
-    private TimeZoneInfo _localTimeZone;
-
-    /// <summary>
-    /// Gets the time which was used as the starting point for the clock in this <see cref="FakeTimeProvider"/>.
-    /// </summary>
-    /// <remarks>
-    /// This can be set by passing in a <see cref="DateTimeOffset"/> to the constructor
-    /// which takes the <c>epoch</c> argument. If the default constructor is used,
-    /// the clock's start time defaults to midnight January 1st 2000.
-    /// </remarks>
-    [Experimental]
-    public DateTimeOffset Epoch { get; } = new DateTimeOffset(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
```

Comment:
I don't think this is a desirable thing, the start time is of extremely limited value since typically it will be out of date instantly. I'd rather not expose a property whose sole use is to read it 2 lines after you have given the value to the constructor.

So I'm going to wait on that until a compelling use case surfaces.

Reply:
Two points:

1. We do not always pass a startTime to the constructor. A factory method or library like AutoFixture could be used to create the FakeTimeProvider.

2. In more complex test scenarios you may want to pass the FakeTimeProvider to methods that move the SUT into a state before the main part of the test method starts executing. Then it is useful if the timeProvider carries around its "epoch"/"start time" since that info will be readily available in these methods, without having to pass that in as an additional argument.

Even in simpler test scenarios, as in tests for this library, having to either specify a `var start = DateTimeOffset.UtcNow` before instantiating `var timeProvider = FakeTimeProvider(start)` or calling `var start = timeProvider.GetUtcNew()` seems like unnecessary work compared to just being able to reference the start time directly from the object that owns it, e.g. `timeProvider.StartTime`.

Reply:
OK, I added StartDateTime. I don't think it's actually useful, but it's trivial and doesn't hurt (hint: kinda like optional auto-advancing time 😜)